src/analysisd/output/prelude.c

```diff
@@ -75,7 +75,7 @@ add_idmef_object(idmef_message_t *msg, const char *object, const char *value)
 
     ret = idmef_path_set(path, msg, val);
     if (ret < 0) {
-        merror("OSSEC2Prelude: IDMEF: Cannot add object '%s': %s.",
+        merror("Wazuh2Prelude: IDMEF: Cannot add object '%s': %s.",
```

Comment:
Shouldn't we also change the name of this in all the Wazuh code? I mean `OSSEC2Prelude` to `Wazuh2Prelude`

Reply:
Thank you! Fixed